src/plugins/intel_gpu/src/graph/activation.cpp

```diff
@@ -27,14 +27,28 @@ layout activation_inst::calc_output_layout(activation_node const& node) {
         activation_func::negative,
         activation_func::negation,
         activation_func::relu,
+        activation_func::relu_negative_slope,
         activation_func::floor,
         activation_func::clamp };
 
-    if (input_node_layout.data_type == data_types::i8 || input_node_layout.data_type == data_types::i32) {
+    if (!data_type_traits::is_floating_point(input_node_layout.data_type)) {
```

Comment:
I suppose it's better to add `in_dt` and to use this one here and in the code below for good readability.